### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by ErebusBat**
- Adds some missing keys to all languages
- Remove `es.signup.error_email_taken` that was previously missed
- Update docs with command to remove keys

### CODE REVIEW

**Comment by ErebusBat** [comment]
[[Authentication] Don't Send Token Reset for X Minutes and Notify Member](https://trello.com/c/DXdYLHDC/163-authentication-dont-send-token-reset-for-x-minutes-and-notify-member)

File: locales/bg.yml [hunk 1]
```diff
@@ -93,6 +93,26 @@ bg:
     return_to_login: връщане към входа
     tab_email: Имейл
     tab_mobile: Телефон
+  groups:
```
Comment (reneklacan) [inline locales/bg.yml]: why is this here too? wasn't this part of Andrii's PR?

Comment (ErebusBat) [reply locales/bg.yml]: @reneklacan He only added `en` translations in fetlife/translations#228 this adds the missing keys to the other languages.
